### DISCUSSION TITLE
Portable App

### DISCUSSION THREAD

**Discussion by REMC19** [Aries85/LightZone#294] [General]
This might be something that should wait until the stable release of 5, but I was wondering if it might be an idea to contact https://portableapps.com/ with a suggestion to create a portable version of LightZone.
Not sure how hard this would be for them to do, or whether you would be happy for them to do it.
They do, however, already do quite a few other photo editing packages (Gimp / darktable / Raw Therapee for example).
Just thinking, it's another way to increase the programs visibility and build the user base.
Also, not sure how much involvement would be needed from yourself (I think they do most of the work - but I couldn't swear to it).

**Comment by ktgw0316** [comment]
Nice idea!  I'll contact them after stable version 5.0.0 release.
